ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py

```diff
@@ -2366,3 +2366,34 @@ def test_torch_logit(
         eps=eps,
         out=None,
     )
+
+
+@handle_frontend_test(
+    fn_tree="torch.frexp",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=1,
+        shared_dtype=True,
+        min_value=-10,
+        max_value=10,
+        min_num_dims=1,
+        max_num_dims=1,
+    ),
+)
+def test_torch_frexp(
+    dtype_and_x,
+    on_device,
+    fn_tree,
+    frontend,
+    test_flags,
+):
+    input_dtype, input = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=input[0],
+        out=None,
+    )
```

Comment:
The test fails for following backends:
1. torch: it fails saying `AttributeError: 'tuple' object has no attribute 'data'` just like torch.max, etc. Decently explained in this discord thread: https://discord.com/channels/799879767196958751/1028707110697058415/1092062095438266458
2. paddle: it fails saying "RecursionError: maximum recursion depth exceeded in comparison" because paddle does not have implementation of frexp